sources/_sector/dex/trades/zkevm/_sources.yml

```diff
@@ -8,4 +8,7 @@ sources:
   - name: pancakeswap_v3_zkevm
     tables:
       - name: PancakeV3Pool_evt_Swap
-      - name: PancakeV3Factory_evt_PoolCreated
\ No newline at end of file
+      - name: PancakeV3Factory_evt_PoolCreated
+  - name: clipper_zkevm
```

Comment:
Hi @ktemo,
seems `clipper_zkevm.ClipperPackedOracleVerifiedExchange_evt_Swapped` is not available/decoded yet.

Do you have the contract address for this one?
We can submit this contract for decoding first.

Reply:
hi, yes I have, this is the contract https://zkevm.polygonscan.com/address/0xe38C90a0233F18749Fb74e595c4De871e5498C13

Reply:
I just submited the contract, I will rebase and trigger a new round of checks here once I see is done, thanks for the heads-up!

Reply:
@Hosuke hi, I just submitted a PR recheck but it failed again with the schema not found error, I checked and the contract is approved and I can find it using the decoded contract tool. I cannot see it in the "New query" library tho, could you help me here, please? is there anything I am missing?. The contract was submitted and approved on August 27th

<img width="1852" alt="Screenshot 2024-09-02 at 10 58 52 a m" src="https://github.com/user-attachments/assets/42d957b0-368a-412a-80ee-0f7a1849c74d">

Reply:
Can you try `select *` from this source table on dune and copy the link here?

Reply:
this is the query from the dune helper:
https://dune.com/queries/1178444/2015988?contract_t6c1ea=0xe38c90a0233f18749fb74e595c4de871e5498c13

If I try to query it directly I also cannot find it

Reply:
here is the `select *` query https://dune.com/queries/4038554

Reply:
I will try to resubmit the contract for decoding again.

Reply:
@Hosuke hi, is there any update on the contract submission/discovery?

Reply:
No, I cannot submit the recontract for decoding.
The contract was decoded.

Though I could not find the decoded source table.

Reply:
What would be the solution here? Should I rebase and try again?

Reply:
You can rebase first.
But we can check the `clipper_zkevm` sources in dune data explorer as well.

Reply:
@Hosuke hi, the job keeps failing, what is your recommendation no move forward with it?

Reply:
actually, I just saw the `clipper_zkevm` available on the dune query dashboard, next try should run with no issues I hope :)

Reply:
i was just stopping by to mention, the table should be ready now! apologies for the delay. there was a minor disruption with the decoding job for this blockchain specifically.